app/qml/pages/MainMenuPage.qml

```diff
@@ -100,7 +100,7 @@ Page {
             ListItem {
                 enabled: root.watch.timeServiceReady
                 ListItemLayout {
-                    title.text: i18n.tr("Synchronize time")
+                    title.text: i18n.tr("Sync time")
```

Comment:
Just for consistency and brevity. It could be misunderstood as a noun instead of a transitive verb.
Possible middleground "**Sync the time**", which one could possible argue should be "**Sync up the time**". which further should be "**Sync clock**" (which avoids the issues up until this point). If one wants to be even more precise from there "**Sync device clock**".